http2_general/test_h2_headers.py

```diff
@@ -650,6 +654,46 @@ def test_trailers_with_continuation_frame_in_request(self):
         self.assertTrue(client.wait_for_response())
         self.assertEqual("200", client.last_response.status, "HTTP response code missmatch.")
 
+    @parameterize.expand(
+        [
+            param(name="end_headers", end_headers=True),
```

Comment:
you should use params here - `param(name="end_headers", flags=["END_HEADERS"])` and then you won't need to use if - `cf_flags = ["END_HEADERS"] if end_headers else ""`